### PR TITLE
New table layout

### PR DESCRIPTION
**Pull request by yte0**

### CODE REVIEW

**Comment by yte0** [comment]
The table lagend look like ordinary buttons and moved to the center of the table
register names removed from indicator rows and then moved to the left in the register rows
table interaction changed in the line chart
axis labels changed to percentages

File: R/shiny_functions.R [hunk 1]
```diff
@@ -42,20 +42,21 @@ qi_table <- function(table_data, selected_units, config) {
   )
   nr_treatment_units <- table_data$treatment_units %>% unique() %>% length()
   col_width <- floor(60 / (nr_treatment_units + 1))
-
+  nr_of_clumns = table_data$treatment_units %>% unique() %>% length() + 2
```
[the remark this answers is not in the document]
Comment (yte0) [reply R/shiny_functions.R:45]: Nope! it is not:)

File: R/shiny_functions.R [hunk 2]
```diff
@@ -183,9 +185,10 @@ indicator_rows <- function(indicator_name, indicator_description, config, datata
       class = "indicator",
       tags$td(
         class = "quality_indicator",
-        tags$div(
-          class = "register_name",
-          tags$h4(reg_name)),
+        style="width: 40%;",
+        # tags$div(
+        #   class = "register_name",
+        #   tags$h4(reg_name)),
```
[the remark this answers is not in the document]
Comment (yte0) [reply R/shiny_functions.R:191]: not yet.

**Comment by arnfinn** [comment]
Please make lintr bot happy, if possible...

**Comment by yte0** [comment]
I will try:)

**Pull request merged**